packages/framework/src/Model/Localization/Localization.php

```diff
@@ -63,7 +71,15 @@ public function getLocale()
      */
     public function getAdminLocale()
     {
-        return 'en';
+        if ($this->allLocales === null) {
+            $this->loadAllLocalesFromAllDomains();
```

Comment:
A better approach imho would be:

```php
$allLocales = $this->getLocalesOfAllDomains();

 if (!in_array($this->adminLocale, $allLocales, true)) {
    throw new AdministrationLocaleNotFoundException($this->adminLocale, $allLocales);
}

return $this->adminLocale;
```

as it doesn't force you to assume that the new method will set values to `$this->allLocales`

Reply:
I thought about it but didn't want to assign into the local variable value that already is in the property.
But it will be much clearer in a way you suggest.

Reply:
The way I see it is that `$this->allLocales` is in fact `string[]|null` but you need `string[]` in your `$allLocales` in your local variable, and `$this->getLocalesOfAllDomains()` can provide it to you.